src/me/StevenLawson/TotalFreedomMod/Commands/Command_oblivion.java

```diff
@@ -0,0 +1,98 @@
+import me.StevenLawson.TotalFreedomMod.TFM_BanManager;
+import me.StevenLawson.TotalFreedomMod.TFM_PlayerList;
+import me.StevenLawson.TotalFreedomMod.TFM_RollbackManager;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+import org.bukkit.util.Vector;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH, blockHostConsole = true)
+@CommandParameters(description = "Obliviates a player", usage = "/<command> <playername>")
+public class Command_oblivion extends TFM_Command
+{
+    @Override
+    public boolean run(final CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (args.length != 1)
+        {
+            return false;
+        }
+
+        final Player player = getPlayer(args[0]);
+
+        if (player == null)
+        {
+            sender.sendMessage(TFM_Command.PLAYER_NOT_FOUND);
+            return true;
+        }
+
+        TFM_Util.adminAction(sender.getName(), "Casting oblivion over " + player.getName(), true);
+        TFM_Util.bcastMsg(player.getName() + " will be completely obliviated!", ChatColor.RED);
+
+        final String ip = player.getAddress().getAddress().getHostAddress().trim();
+
+        // remove from whitelist
+        player.setWhitelisted(false);
+
+        // deop
+        player.setOp(false);
+
+        // ban IPs
+        for (String playerIp : TFM_PlayerList.getEntry(player).getIps())
+        {
+            TFM_BanManager.addIpBan(new TFM_Ban(playerIp, player.getName()));
+        }
+
+        // ban uuid
+        TFM_BanManager.addUuidBan(player);
+
+        // set gamemode to survival
+        player.setGameMode(GameMode.SURVIVAL);
+
+        // clear inventory
+        player.closeInventory();
+        player.getInventory().clear();
+
+        // rollback
+        TFM_RollbackManager.rollback(player.getName());
+
+        // Shoot the player in the sky
+        player.setVelocity(player.getVelocity().clone().add(new Vector(0, 20, 0)));
+
+        new BukkitRunnable()
+        {
+            @Override
+            public void run()
+            {
+                // strike lightning
+                player.getWorld().strikeLightning(player.getLocation());
+
+                // kill (if not done already)
+                player.setHealth(0.0);
+            }
+        }.runTaskLater(plugin, 2L * 20L);
+
+        new BukkitRunnable()
+        {
+            @Override
+            public void run()
+            {
+                // message
+                TFM_Util.adminAction(sender.getName(), "Banning " + player.getName() + ", IP: " + ip, true);
+
+                // kick player
+                player.kickPlayer(ChatColor.RED + "FUCKOFF, and get your shit together!");
```

Comment:
You may want to change this to:

player.kickPlayer(ChatColor.RED + "You have been obliterated by:" + " " + sender.getName() + "!");